tests/typings/webdriverio/async.ts

```diff
@@ -86,7 +86,7 @@ async function bar() {
 
     const executeResult = await browser.execute(function (x: number) {
         return x
-    }, 4)
+    }, 4) as number
```

Comment:
the additional cast should not be required!
![image](https://user-images.githubusercontent.com/25589559/97417136-9cbc8c80-1907-11eb-96ee-9e151b69ba0d.png)